include/picongpu/fields/MaxwellSolver/YeePML/YeePML.hpp

```diff
@@ -412,10 +413,16 @@ namespace picongpu
                      * to-be-absorbed components.
                      */
                     solver.template updateBSecondHalf<CORE + BORDER>(currentStep);
+                    auto incidentFieldSolver = fields::incidentField::Solver{cellDescription};
```

Comment:
Why is `incidentFieldSolver` not stored as a member of `YeePML`?

Reply:
It can be, would be nothing wrong with that. I think i did it forst in the `Yee` solver similar to how `LaserPhysics` object is created in-place.